code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -366,4 +366,196 @@
 	damage = 6	//they're huge! they should do a little more damage, i'd even go for 15-20 maybe...
 
 /datum/species/proc/handle_can_equip(obj/item/I, slot, disable_warning = 0, mob/living/carbon/human/user)
-	return 0
\ No newline at end of file
+	return 0
+
+/datum/species/proc/handle_vision(mob/living/carbon/human/H)
```

Comment:
@KasparoVy It opens up possibilities- ex, species can actually control the vision of their humans easily.